client/INLClient.py

```diff
@@ -46,10 +52,18 @@ def check_server(self, server):
         getter = JobGetter(self.client_info)
         claimed = getter.find_job()
         if claimed:
+            if self.get_client_info('manage_build_root'):
+                self.create_build_root()
+
             load_modules = settings.CONFIG_MODULES[claimed['config']]
             os.environ["CIVET_LOADED_MODULES"] = ' '.join(load_modules)
             self.modules.clear_and_load(load_modules)
             self.run_claimed_job(server[0], [ s[0] for s in settings.SERVERS ], claimed)
+            self.client_info["jobs_ran"] += 1
+
+            if self.get_client_info('manage_build_root'):
+                self.remove_build_root()
+
```

Comment:
These lines are, more or less, the entire reason for this PR right? - Just want to explicitly create/remove any build_root dir specified in the env rather than assume that it occurs external to the client?

Reply:
Correct.

The additional checks (on init, beginning of the poll loop, exit of the poll loop) are for when things go bad (graceful quits, cancel quits, etc) and they are also all tested